.travis.yml

```diff
@@ -112,6 +115,14 @@ script:
     - $CXX --version
     - if [ -z "$TRAVIS_TAG" ]; then cargo build --verbose; fi
     - if [ -z "$TRAVIS_TAG" ]; then cargo test -- --nocapture; fi
+    - |
+      if [ -z "$COVERAGE" ]; then
+        cargo clean;
+        CXXFLAGS="--coverage" LDFLAGS="--coverage -lgcov" CARGO_INCREMENTAL=0 RUSTFLAGS="-Zprofile -Ccodegen-units=1" cargo +nightly test;
+        zip -0 ccov.zip `find . \( -name "grcov*.gc*" -o -name "llvmgcov.gc*" \) -print`;
+        ./target/debug/grcov ccov.zip -s src/ -t lcov --llvm --branch --ignore-dir libstd/* --ignore-dir libcore/* --ignore-dir liballoc/* --ignore-dir libtest/* --ignore-dir "<*" --ignore-dir "path_rewriting.rs" > lcov.info;
```

Comment:
Why do we need to ignore `path_rewriting.rs`?

Reply:
It's strange here, in fact one gcno contains the path path_rewriting.rs and src/path_rewriting.rs.
So the latter is correct but not the former.

Reply:
Weird, maybe it's worth filing another bug in the Rust repository about this.